VideoLocker/src/main/java/org/edx/mobile/view/view_holders/AuthorLayoutViewHolder.java

```diff
@@ -1,16 +1,110 @@
 package org.edx.mobile.view.view_holders;
 
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.widget.TextViewCompat;
 import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
 import android.widget.TextView;
 
+import com.bumptech.glide.Glide;
+import com.google.inject.Inject;
+import com.joanzapata.iconify.IconDrawable;
+import com.joanzapata.iconify.fonts.FontAwesomeIcons;
+
 import org.edx.mobile.R;
+import org.edx.mobile.discussion.DiscussionTextUtils;
+import org.edx.mobile.discussion.IAuthorData;
+import org.edx.mobile.model.api.ProfileModel;
+import org.edx.mobile.module.prefs.UserPrefs;
+import org.edx.mobile.user.ProfileImage;
+import org.edx.mobile.util.Config;
+
+import roboguice.RoboGuice;
 
 public class AuthorLayoutViewHolder {
 
-    public final TextView discussionAuthorTextView;
+    public interface ProfileImageProvider {
```

Comment:
This interface doesn't really belong here, since it's conceptually a generic interface that can be implemented by any discussion model. The fact that it's only being made use of by this view holder class is an implementation detail. As such, it should ideally be defined in the same package as the models themselves.